dask/dataframe/tests/test_groupby.py

```diff
@@ -1506,108 +1784,134 @@ def agg_mode(s):
-@pytest.mark.parametrize('min_count', [0, 1, 2, 3])
+@pytest.mark.skipif(
+    PANDAS_VERSION < "0.22.0",
+    reason="Parameter min_count not implemented in "
+    "DataFrame.groupby().sum() and DataFrame.groupby().prod()",
+)
+@pytest.mark.parametrize("min_count", [0, 1, 2, 3])
 def test_with_min_count(min_count):
-    dfs = [pd.DataFrame({
-                        'group':['A', 'A', 'B'],
-                        'val1':[np.nan, 2, 3],
-                        'val2':[np.nan, 5, 6],
-                        'val3':[5, 4, 9]
-                        }),
-           pd.DataFrame({
-                        'group':['A', 'A', 'B'],
-                        'val1':[2, np.nan, np.nan],
-                        'val2':[np.nan, 5, 6],
-                        'val3':[5, 4, 9]
-                        })]
+    dfs = [
+        pd.DataFrame(
+            {
+                "group": ["A", "A", "B"],
+                "val1": [np.nan, 2, 3],
+                "val2": [np.nan, 5, 6],
+                "val3": [5, 4, 9],
+            }
+        ),
+        pd.DataFrame(
+            {
+                "group": ["A", "A", "B"],
+                "val1": [2, np.nan, np.nan],
+                "val2": [np.nan, 5, 6],
+                "val3": [5, 4, 9],
+            }
+        ),
+    ]
     ddfs = [dd.from_pandas(df, npartitions=4) for df in dfs]
 
     for df, ddf in zip(dfs, ddfs):
-        assert_eq(df.groupby('group').sum(min_count=min_count),
-                  ddf.groupby('group').sum(min_count=min_count))
-        assert_eq(df.groupby('group').prod(min_count=min_count),
-                  ddf.groupby('group').prod(min_count=min_count))
+        assert_eq(
+            df.groupby("group").sum(min_count=min_count),
+            ddf.groupby("group").sum(min_count=min_count),
+        )
+        assert_eq(
+            df.groupby("group").prod(min_count=min_count),
+            ddf.groupby("group").prod(min_count=min_count),
+        )
 
 
 def test_groupby_group_keys():
-    df = pd.DataFrame({
-        'a': [1, 2, 2, 3],
-        'b': [2, 3, 4, 5],
-    })
-    ddf = dd.from_pandas(df, npartitions=2).set_index('a')
-    pdf = df.set_index('a')
+    df = pd.DataFrame({"a": [1, 2, 2, 3], "b": [2, 3, 4, 5]})
+    ddf = dd.from_pandas(df, npartitions=2).set_index("a")
+    pdf = df.set_index("a")
 
     func = lambda g: g.copy()
-    expected = pdf.groupby('a').apply(func)
-    assert_eq(expected, ddf.groupby('a').apply(func, meta=expected))
+    expected = pdf.groupby("a").apply(func)
+    assert_eq(expected, ddf.groupby("a").apply(func, meta=expected))
+
+    expected = pdf.groupby("a", group_keys=False).apply(func)
+    assert_eq(expected, ddf.groupby("a", group_keys=False).apply(func, meta=expected))
+
 
-    expected = pdf.groupby('a', group_keys=False).apply(func)
-    assert_eq(expected, ddf.groupby('a', group_keys=False).apply(func, meta=expected))
+def test_groupby_cov():
```

Comment:
I'll probably remove this test -- but wanted it during interim for easy testing

Reply:
I think it's worth keeping.

Reply:
ok